analyzer/src/main/kotlin/managers/SpdxDocumentFile.kt

```diff
@@ -361,7 +361,7 @@ class SpdxDocumentFile(
         val packageFile = definitionFile.parentFile.resolve(spdxPackage.packageFilename)
 
         if (packageFile.isFile) {
-            val managedFiles = findManagedFiles(packageFile.parentFile, ALL)
+            val managedFiles = findManagedFiles(packageFile.parentFile)
```

Comment:
IIRC the usage of `ALL` was deliberate here to make sure that an error is created if a package manager dependency was defined but the required package manager is disabled.

Reply:
I'm not sure what you mean... `ALL` is the default argument, and it always returns all package manager implementations from the classpath, no matter whether they were disabled at the CLI or not.

Reply:
What I meant is it was deliberate to be explicit following the idea that:
* If you don't care about the value of an argument, rely on the default value.
* If you care about the value of an argument, be explicit and do not rely on the default value to never change.

Reply:
But I mainly wanted to mention it, I'll leave it up to you if you want to change it or not as it does not break anything.

Reply:
I believe there's a third idea to default arguments in general, which is to not repeat arguments that barely need to be changed. So you *do* care about the argument, *and* you're fine with the default value. From that perspective I'd prefer to keep the change, esp. as I cannot envision us to ever change the default value for the `packageManagers` argument to anything else than all package managers.